Author: Michael R. Crusoe <michael.crusoe@gmail.com>
Description: None of the licenese require source redistribution

So mark the package as such.
--- capstone-sys.orig/build.rs
+++ capstone-sys/build.rs
@@ -361,4 +361,5 @@
         copy(&pregenerated_bindgen_impl, &out_impl_path)
             .expect("Unable to update capstone bindings");
     }
+    println!("dh-cargo:deb-built-using=capstone=0={}", env::var("CARGO_MANIFEST_DIR").unwrap());
 }
